src/controller.cpp

```diff
@@ -44,15 +44,16 @@ tf2::Transform to_transform(const geometry_msgs::Pose & pose)
 }
 
 // Returns the square distance between two points
-double distSquared(const tf2::Transform & a, const tf2::Transform & b)
+units::distance_squared_t distSquared(const tf2::Transform & a, const tf2::Transform & b)
```

Comment:
I think `distance_squared_t` is just as explicit as `units::distance_squared_t`.

(Also holds for other units with `units::` prepend.